test/util/networkAttachmentDefinition.go

```diff
@@ -0,0 +1,88 @@
+package util
```

Comment:
any reason to title the file with capital A(ttachment)  + D(efinition)?

Reply:
I used to name files with camel case notation, like variables in code. Will change it to lowercase.